products/compute/api.yaml

```diff
@@ -4835,6 +4835,120 @@ objects:
+    identity:
+      - ipAddress
+      - port
+      - instance
+    nested_query: !ruby/object:Api::Resource::NestedQuery
+      keys:
+        - items
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'https://cloud.google.com/load-balancing/docs/negs/'
+      api: 'https://cloud.google.com/compute/docs/reference/rest/beta/networkEndpointGroups'
+    async: !ruby/object:Api::Async
+      operation: !ruby/object:Api::Async::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/zones/{{zone}}/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::Async::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::Async::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::Async::Error
+        path: 'error/errors'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'zone'
+        resource: 'Zone'
+        imports: 'name'
+        description: |
+          Zone where the containing network endpoint group is located.
+        required: true
+        url_param_only: true
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'networkEndpointGroup'
+        resource: 'NetworkEndpointGroup'
+        imports: 'name'
+        description: |
+          The network endpoint group this endpoint is part of.
+        required: true
+        url_param_only: true
+    properties:
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'instance'
+        resource: 'Instance'
+        imports: 'name'
+        description: |
+          The name for a specific VM instance that the IP address belongs to.
+          This is required for network endpoints of type GCE_VM_IP_PORT.
+          The instance must be in the same zone of network endpoint group.
+        required: true
+      - !ruby/object:Api::Type::Integer
+        name: 'port'
+        description: |
+          Optional port number of network endpoint. If not specified and the
+          NetworkEndpointGroup.network_endpoint_type is GCE_IP_PORT,
+          the defaultPort for the network endpoint group will be used.
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'ipAddress'
+        description: |
+          Optional IPv4 address of network endpoint. The IP address must belong
+          to a VM in GCE (either the primary IP or as part of an aliased IP
+          range). If the IP address is not specified, then the primary IP
+          address for the VM instance in the network that the network endpoint
+          group belongs to will be used.
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'healthBackendService'
```

Comment:
Where do these come from? I can't see them on https://cloud.google.com/compute/docs/reference/rest/beta/networkEndpointGroups/get

Reply:
these are networkEndpoints, not group https://cloud.google.com/compute/docs/reference/rest/v1/networkEndpointGroups/listNetworkEndpoints

Reply:
🤦‍♀

Reply:
Do we expect to always only have a single entry in `healths`? I can't find documentation one way or another.

Reply:
I feel like I read either in gcloud or the actual proto that the object description was "URL of the health $resource" but I also might just remove them - either I missed this originally or it was recently updated to not show health checks by default (you have to provide "show health checks", and I'd rather not add them for now if we're not sure what kind of format they will be in the future.

Reply:
Removing sgtm- the schema is unclear, and a user request will provide more information about what we need to support.